pkg/common/config/types.go

```diff
@@ -81,6 +81,14 @@ type VirtualCenterConfig struct {
 	User string `gcfg:"user"`
 	// vCenter password in clear text.
 	Password string `gcfg:"password"`
+	// ConnectionLabel (intentionally not exposed via the config) is a unique label to
+	// be used in place of the vcServer as the primary connection key. If one label is set,
+	// all virtual center configs must have a unique label.
+	ConnectionLabel string
```

Comment:
Wondering if `ConnectionLabel` is still the best name for this field since it's just a unique reference for a vCenter? A few off the top of my head:
* ID 
* Name 
* Key
* ConnectionKey

Reply:
I think at a high level it is. Also, from an end-user point of view, it is as well. Referring to it as any of those things doesn't have real meaning to an end-user.

Having said that, internally in code, yes it's a key. I don't care either way what the variable is called, but in the documentation, I think it most definitely should be a label. For understandability, if the docs say label, do we want to be consistent?

Reply:
Good point, I'm okay to merge as `ConnectionLabel` but when we get to docs I think it's worth another discussion to see if there's a better name for this.

Reply:
If you still want to change in code, better to do it now. Tell me what you want to use.

Reply:
Updating to TenantRef